libsearch/util.go

```diff
@@ -80,3 +82,12 @@ func WriteFileAtomic(pathname string, content []byte) error {
 	err = os.Rename(tmpFile.Name(), pathname)
 	return err
 }
+
+// NormalizeKeyword normalizes a keyword for adding into the index by converting
+// it to lower case and keeping only the alphanumeric characters.
+func NormalizeKeyword(keyword string) string {
+	lowerKeyword := strings.ToLower(keyword)
+
+	re := regexp.MustCompile("[^a-z0-9]")
```

Comment:
Oh to do that we then need to view the keyword as unicode encoding, then we can scan rune by rune and remove the invalid ones. This should be do-able.